axelrod/tests/strategies/test_better_and_better.py

```diff
@@ -23,5 +23,14 @@ class TestBetterAndBetter(TestPlayer):
     def test_strategy(self):
         """Tests that the strategy gives expected behaviour."""
         self.first_play_test(D, seed=3)  # D is very unlikely
```

Comment:
that's actually a bit tough.  on the first round, the chance is 0.001.  here's the problem.
```
def test_for_first_round_cooperations(self):
        for seed in range(2000):
            try:
                self.first_play_test(D, seed=seed)
            except AssertionError:
                print('failed at {}'.format(seed))
```
the output is 
```
......failed at 1514
failed at 1651
.......
----------------------------------------------------------------------
Ran 13 tests in 0.119s

```
note the time it takes.  without this it runs in 0.008s .  

that's why i included the following test.  I thought it nicely visualized getting better and better on the scale of probability (which increases 0.001 per round so is 0.2 by round 200)
```
        for index in range(200):
            if index in [64, 79, 91, 99, 100, 107, 111, 119, 124, 127, 137, 141, 144, 154, 192, 196]:
                actions.append((C, D))
            else:
                actions.append((D, D))
        self.versus_test(axelrod.Defector(), expected_actions=actions, seed=8)
```